jupyterhub/app.py

```diff
@@ -1151,7 +1158,14 @@ def user_stopped(user):
             self.users[orm_user.id] = user = User(orm_user, self.tornado_settings)
             self.log.debug("Loading state for %s from db", user.name)
             spawner = user.spawner
-            status = yield spawner.poll()
+            status = 0
+            if user.server:
+                try:
+                    status = yield spawner.poll()
+                except Exception:
+                    self.log.exception("Failed to poll Spawner for %s, assuming it is not running.", user.name)
```

Comment:
Just to clarify what is happening here and meaning of `it`:

"Failed to poll Spawner for %s, assuming Spawner is not running."